source/security/network.txt

```diff
@@ -0,0 +1,490 @@
+of the following IP addresses:
+
+.. code-block:: text
+   :caption: Azure outbound IP addresses
+
+   20.105.25.17
+   20.212.99.191
+   20.24.112.135
+   20.53.104.226
+   20.84.232.59
+   20.96.47.95
+   40.112.209.0
+   52.149.111.83
+
+GCP
+```
+
+Download GCP IP Addresses: `JSON <https://www.mongodb.com/docs/realm-sdks/mongodb/AtlasAppServicesPublicIPsGCP.json>`__, `CSV <https://www.mongodb.com/docs/realm-sdks/mongodb/AtlasAppServicesPublicIPsGCP.csv>`__
+
+Outbound requests from an app deployed to GCP will originate from one of
+the following IP addresses:
+
+.. code-block:: text
+   :caption: GCP outbound IP addresses
+
+   34.150.239.218
+   34.69.118.121
+   34.78.133.163
+   34.82.246.143
+   34.93.58.231
+
+.. note::
+
+   The above IP lists only apply to outgoing requests from Atlas Functions,
+   including triggers and HTTPS endpoints that make outgoing requests.
+
+   For requests that originate from the Sync server, we recommend allowlisting
+   the entire subnet of the App's Deployment Region and cloud provider. You can
+   find the Deployment Region in the App Services UI under :guilabel:`App Settings >
+   General > Deployment Region`.
+
+.. _security-dns-filtering-sync-client-to-sync-server:
+
+Communicate from the Client to the Sync Server
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When you use Device Sync, you can use DNS filtering to allow connections
+from the Sync client to the Sync server. Using DNS filtering, you can access 
+``*.realm.mongodb.com`` via HTTPS or port 443.
+
+.. _security-allowed-request-origins:
+
+Allowed Request Origins
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You can define this :ref:`configuration option <appconfig-app>` in the 
+app-level ``realm_config.json`` file. This field accepts an array of 
+URLs that incoming requests may originate from. If you define any allowed 
+request origins, then App Services blocks any incoming request from 
+an origin that is not listed.
+
+.. _security-ip-access-list:
+.. _ip-access-list:
+
+IP Access List
+~~~~~~~~~~~~~~
+
+App Services allows client requests from the enabled entries in the app's
+:ref:`IP access list <manage-ip-access>`. Allowed requests will still use 
+App Services's authentication and authorization rules. When you add IP 
+access list entries, App Services blocks any request originating from 
+an IP that is not on the access list.
+
+.. important::
+
+   By default, any newly-created App allows access from any client IP
+   by adding an access list entry for 0.0.0.0/0. If you delete this entry, 
+   no client can access your App from any IP address. 
+
+.. _security-api-access-list:
+
+API Access List
+~~~~~~~~~~~~~~~
+
+When you :ref:`create an API key <cli-auth-with-api-token>`, you can specify 
+IP addresses that can use this API key to access your project or organization. 
+If you specify an IP address, App Services blocks any request originating 
+from an IP address that is not on the access list.
```

Comment:
Can we make it clearer here that this is for Atlas API key pairs that you use with Realm CLI/the Admin API (i.e. not the API key auth provider) and link to the Atlas docs instead of a realm-cli procedure?